tests/test_lime_survey.py

```diff
@@ -254,6 +259,33 @@ def test_supervision_transformation_questions(self):
             survey.responses.iloc[6:9, -4:], ref, msg="DataFrames not equal."
         )
 
+    def test_phd_duration_transformation_questions(self):
```

Comment:
Can you please also add a similar test to `test_lime_transformations.py`? Can basically copy paste this one and set up a new test class like the other transformations there.

Reply:
Sorry for the delay Dawei, I have committed the test for transformation. Please let me know what you think. :)

Reply:
Update your main branch with `git pull` and then do a rebase with `git rebase main` because new changes have been merged into main recently.

Reply:
I did as you suggested, but I'm not sure if its okay now... maybe you can help me later in the weekly meeting?